containers/podman-desktop-extension-bootc
# feat: adds webview build page

feat: adds webview build page

### What does this PR do?

* Adds a webview build page
* Uses the last deployed bootc build as the "previous selection" for the
  page
* Added tests for the webview
* Fixed current tests identified when adding new tsconfig

### Screenshot / video of UI



https://github.com/containers/podman-desktop-extension-bootc/assets/6422176/ccbbbdcf-9cae-42e3-a12b-83badce23d66




### What issues does this PR fix or reference?



Closes https://github.com/containers/podman-desktop-extension-bootc/issues/141
Closes https://github.com/containers/podman-desktop-extension-bootc/issues/155

### How to test this PR?



1. `yarn watch` in this extension directory
2. `cd podman-desktop`
3. `yarn watch --extension-folder ../bootc/packages/backend`
4. Press the bootc icon on the left that should be appear, and start a
   build via the shown webview.
